src/main/java/org/jenkinsci/plugins/github_branch_source/Endpoint.java

```diff
@@ -129,9 +129,9 @@ public int hashCode() {
     }
 
     @Extension
-    public static class DesciptorImpl extends Descriptor<Endpoint> {
+    public static class DescriptorImpl extends Descriptor<Endpoint> {
```

Comment:
AFAIK renaming this is not an issue as it's not used in any persisted configuration. Please double check.

Reply:
https://github.com/search?q=DesciptorImpl&type=Code 😄

Reply:
Looks good to me.  `Endpoint` is persisted, but this class is only instantiated when constructing that class, but otherwise isn't used.

Reply:
Yeah, I think it's ok, it's not a top-level descriptor so I don't think the name of the descriptor is used in any serialized data. That said, it's `public`, so the change _could_ break some non-public code that wasn't doing anything wrong, and fixing the typo doesn't have any real benefit, so I would consider leaving this unchanged (just thinking extra conservatively).